objects-api/src/main/kotlin/com/ritense/objectsapi/taak/TaakObjectConnector.kt

```diff
@@ -128,9 +128,19 @@ class TaakObjectConnector(
         formulierId: String?,
         formulierUrl: String?,
     ): TaakObjectDto {
+        val bsn = bsnProvider?.getBurgerServiceNummer(task)
+        val kvk = kvkProvider?.getKvkNummer(task)
+        val identificaties = mutableListOf<TaakIdentificatie>()
+        if (bsn != null) {
+            identificaties += TaakIdentificatie("bsn", bsn)
+        }
+        if (kvk != null) {
+            identificaties += TaakIdentificatie("kvk", kvk)
+        }
         val taakObject = TaakObjectDto(
```

Comment:
Do we still want do continue even if there's no kvk or bsn?

Reply:
No, I've added an exception